src/strings/en-us.json

```diff
@@ -359,7 +360,6 @@
     "HeaderApiKeysHelp": "External applications are required to have an API key in order to communicate with the server. Keys are issued by logging in with a normal user account or manually granting the application a key.",
     "HeaderApp": "App",
     "HeaderAppearsOn": "Appears On",
-    "HeaderAudioBooks": "Audio Books",
```

Comment:
Can we avoid these translation key changes? Translators would need to re-translate them for all languages. :sweat_smile: